plugins/packageManager/controller.xql

```diff
@@ -1,4 +1,4 @@
-xquery version "1.0";
+xquery version "3.1";
```

Comment:
OK. I re-used code from the main controller.xql, that already uses xquery version 3.0. So is 3.0 safe?
Otherwise we should definitely up the minimum requirement.